.prettierrc.js

```diff
@@ -1 +1,10 @@
-module.exports = require('kcd-scripts/prettier')
+const prettierConfig = require("kcd-scripts/prettier");
+
+module.exports = {
+  ...prettierConfig,
+  arrowParens: "always",
+  bracketSpacing: true,
+  semi: true,
+  singleQuote: false,
+  trailingComma: "es5",
+};
```

Comment:
sorry to be a pain, but what happens if we just completely delete this file?  it should just then use the prettier defaults no?

Reply:
I've overwritten the `kcd-scripts/prettier` config with all changed values, just so it's clear that we're not using those values.
Another option would be to just return an empty object (`{}`) and that would also use the default prettier config.

Not providing a config, will make `kcd-scripts` use the built-in config like you can see here:

https://github.com/kentcdodds/kcd-scripts/blob/8899fe9624a4358807ccb866ca9f2b959cf6bf56/src/scripts/format.js#L12-L16

I also wanted to have the file in the repo, so that IDE's can pick up everything too.

Reply:
Ok then let's have it return an empty object please.

Reply:
Done 🙂